commitizen/cli.py

```diff
@@ -290,6 +309,28 @@ def commitizen_excepthook(type, value, tracekback, debug=False):
 sys.excepthook = commitizen_excepthook
 
 
+def parse_no_raise(comma_separated_no_raise: str) -> List[int]:
+    """Convert the given string to exit codes.
+
+    Receives digits and strings and outputs the parsed integer which
+    represents the exit code found in exceptions.
+    """
+    no_raise_items = comma_separated_no_raise.split(",")
+    no_raise_codes = []
+    for item in no_raise_items:
+        if item.isdecimal():
+            no_raise_codes.append(int(item))
+            continue
+        try:
+            exit_code = ExitCode[item]
+        except KeyError:
+            out.warn(f"WARN: no_raise key {item} does not exist. Skipping.")
```

Comment:
nitpick: do we want to quote `{item}` (i.e., "WARN: no_raise key '{item}' does not exist. Skipping") to avoid user pass in a string with space